starter/script.js

```diff
@@ -1 +1,135 @@
-// put code here!
+var buffer = function(){
+  console.log("***********************")
+}
+
+console.log("You need to get into the non-descript gray building \n to turn in your form. You are greeted by a security guard.")
+buffer()
+var theGuard = prompt("Please empty your pockets and wait to be frisked.  Do you comply? (please enter 'y' or 'n')") === 'y'
+
+if (theGuard) {
+  console.log("You empty your pockets and are frisked.  The guard finds a switchblade in your pocket.")
+buffer()
+  var switchBlade = prompt("Do you attempt to use the blade on the guard? (please enter 'y' or 'n')");
+
+    if (switchBlade === 'y') {
+      console.log("You take a lunging stab at the guard. You miss. He shoots you. You die. You have failed The Bureaucracy.");
+      buffer()
+    }
+    if (switchBlade === 'n') {
+      console.log("The guard confiscates your knife and allows you to enter. You proceed to the elevator.");
+      console.log("You are greeted at the elevator by a tiny old man.");
+      buffer()
+      var oldMan = prompt("Please tell the old man what floor you wish to go to (number 1-100)");
+
+        if (oldMan <= 12) {
```

Comment:
this line shouldn't be indented the extra level, it should be at the same level as the previous line.